test/verify/check-users

```diff
@@ -149,16 +149,16 @@ class TestAccounts(MachineCase):
         b.wait_in_text('#accounts-list', "Berta Bestimmt")
 
         # Check home directory
-        home_dir = m.execute("getent passwd berta | cut -f6 -d:").strip()
-        self.assertTrue(home_dir.endswith("/berta"))
-        self.assertEqual(m.execute(f"stat -c '%U' {home_dir}").strip(), "berta")
+        home_dir = m.execute("getent passwd Berta | cut -f6 -d:").strip()
```

Comment:
Eww, we support users/groups with upper case names? I wasn't even aware that Linux supports that. It technically works (at least sometimes), but the [manpage](https://www.unix.com/man-page/linux/8/useradd/) recommends using only lowercase names. Changing the test above to type in "Berta" is a good test to cover this unusual feature on all OSes -- but please also retain a test case with a standard lower-case name, to make sure this works as well.

Reply:
Yes, I know it's users (although the question applies to groups as well). Well, as long as it works everywhere, fine for me.